client/src/plugins/version-info/VersionInfoOverlay.js

```diff
@@ -28,28 +28,40 @@ export function VersionInfoOverlay(props) {
       id="version-info-overlay" anchor={ props.anchor } onClose={ props.onClose } offset={ OFFSET }
       className={ css.VersionInfoOverlay }
     >
-      <Overlay.Title>
+      <WhatsNewSection version={ props.version } />
+
+      <LearnMoreSection />
+    </Overlay>
+  );
+}
+
+function WhatsNewSection(props) {
+
+  return (
+    <Section maxHeight="calc(100vh - 250px)">
+      <Section.Header>
```

Comment:
I'm wondering why codecov is complaining here (for the feedback), although we have rendering tests for these. Probably because [we use `shallow`](https://github.com/camunda/camunda-modeler/blob/develop/client/src/plugins/version-info/__tests__/VersionInfoOverlaySpec.js#L38)?